core/imageroot/usr/local/agent/pypkg/agent/smarthost.py

```diff
@@ -0,0 +1,63 @@
+#
+# Copyright (C) 2021 Nethesis S.r.l.
+# http://www.nethesis.it - nethserver@nethesis.it
+#
+# This script is part of NethServer.
+#
+# NethServer is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License,
+# or any later version.
+#
+# NethServer is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NethServer.  If not, see COPYING.
+#
+
+import redis
+import os
+import json
+
+class SmartHost:
+
+    def load_smtp(self):
+        rdb = self._redis_connect()
+        conf = rdb.hgetall('cluster/smarthost')
+
+        if not conf:
+            data={
+                "port": 587,
+                "host": "",
+                "username":"",
+                "password": "",
+                "enabled": False,
+                "tls": True,
+                "tls_verify": True
+            }
+        else:
+            data={
+                "port": int(conf['port']),
+                "host": conf['host'],
+                "username": conf['username'],
+                "password": conf['password'],
+                "enabled": conf['enabled'] == "on",
```

Comment:
Sorry for giving you a bad direction, but after looking at [the docs](https://github.com/NethServer/ns8-core/pull/262/files#diff-f01cd79b3f5c1eb444fd7ad38a2ccbd663c1847a9c06c5340d67e9c4dbb96915R133) a recall a conversation with @gsanchietti about storing boolean values in Redis as `"1" "0"` strings.

Just cosmetic but relevant to establish core code conventions.

Reply:
done